appdb.py

```diff
@@ -15,19 +15,32 @@
 mysql=MySQL(app)
 
 @app.route('/', methods=['GET','POST'])
+def home():
+    if request.method=='GET':
+        return render_template('home.html')
+    if request.method=='POST':
+        return render_template('home.html')
+
+@app.route('/newuser')
+def success():
+    return render_template('success.html')
+
+
+@app.route('/login', methods=['GET','POST'])

```

Comment:
Maybe this should be called `/register` since we are yet to add the logging in itself.

Reply:
Okay so maybe some clarification is needed here:

### Registering
Creating a new account by inserting user data into the database. This is what you're doing here in this function (so your page should be `/register` and the `/success` page should say that you've successfully registered).

### Logging in
Enable the user to "enter" their account. What this means is that they enter their details (username, password, etc) and you check if those details match the ones **already in your database**. If they do, you grant them access to their account.
**So what does granting access mean?** In this case it's assigning a variable in the `session` that states that the user is logged in (and that when logged out, is changed) for example.  See the code [here](https://pythonspot.com/login-authentication-with-flask/) for example (read the code to see where this is), and read more about Flask sessions [here](https://www.tutorialspoint.com/flask/flask_sessions.htm).

For now, it is enough to register the users (without logging in or out :)) and the login part can be done separately after we merge this pull request.